README.md

```diff
@@ -37,6 +37,19 @@ Open up [localhost:3000](http://localhost:3000) and start clicking around.
 
 Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.
 
+### Running tests
+
+Fonctional tests are located in `cypress` folder. Prior to running test, you'll need to install [cypress](https://www.cypress.io/) globally:
```

Comment:
`Fonctional` should be changed to `Functional`. We also shouldn't be recommending a global install of cypress, but a local one.

Reply:
Also `located in` should be `located in the` and `running test` changed to `running tests`.